fwbot/tests/test_merge_request.py

```diff
@@ -0,0 +1,113 @@
+from textwrap import dedent
+from unittest.mock import Mock, patch
+
+from ..api import ForwardbotGitlabClient
+from .common import ForwardBotCase, MockResponse
+
+
+class TestMergeRequest(ForwardBotCase):
+    def test_obtain_from_event_create(self):
```

Comment:
Numerate tests, e.g.:

```suggestion
    def test_01_obtain_from_event_create(self):
```